modules/signing_worker/manifests/init.pp

```diff
@@ -210,37 +213,41 @@
         require     => [Vcsrepo[$scriptworker_scripts_clone_dir], Python::Virtualenv["signingworker_${user}"]],
     }
 
-    # We only clone this once for three reasons:
-    # 1) It is almost never updated
-    # 2) We don't support general code deployments through puppet (yet)
-    # 3) The clone url contains a github token, which we don't want sitting around on disk
-    #
-    # In an ideal world we'd still use `vcsrepo` for this, but it breaks after we
-    # clean up the token, so we're stuck with this for now.
-    exec { "clone widevine ${scriptworker_base}":
-        command => "git clone https://${widevine_user}:${widevine_key}@github.com/mozilla-services/widevine ${widevine_clone_dir}",
-        user    => $user,
-        group   => $group,
-        unless  => "test -d ${widevine_clone_dir}",
-        path    => ['/bin', '/usr/bin'],
-        require => File[$scriptworker_base],
-    }
-    # This has credentials in it. Clean up.
-    ->file { "Remove widevine directory ${scriptworker_base}":
-        ensure  => absent,
-        path    => "${widevine_clone_dir}/.git",
-        recurse => true,
-        purge   => true,
-        force   => true,
-    }
-    exec { "install ${scriptworker_base} widevine":
-        command     => "${virtualenv_dir}/bin/python setup.py install",
-        cwd         => $widevine_clone_dir,
-        user        => $user,
-        group       => $group,
-        refreshonly => true,
-        subscribe   => [Exec["clone widevine ${scriptworker_base}"], Python::Virtualenv["signingworker_${user}"]],
-        require     => [Exec["clone widevine ${scriptworker_base}"], Python::Virtualenv["signingworker_${user}"]],
+    if $widevine_filename {
```

Comment:
I changed it so we check if `$widevine_filename` is set instead of only doing it for ff role.
Once we confirm [or not] that TB doesn't need it, we can just delete the config.